go.mod

```diff
@@ -137,5 +135,6 @@ replace (
 	github.com/cosmos/admin-module => github.com/Ethernal-Tech/admin-module v0.0.0-20221102105340-e693f4d379c3
 	github.com/cosmos/ibc-go/v3 => github.com/informalsystems/ibc-go/v3 v3.0.0-beta1.0.20220816140824-aba9c2f2b943
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/osmosis-labs/osmosis/v10 v10.2.0 => github.com/neutron-org/osmosis/v10 v10.2.0-native-sdk-4
```

Comment:
I think we need a separate branch in our osmosis fork (like we did with wasmd fork) where we will push necessary changes for us.
Also it's easier to see the diff

Reply:
done